package.json

```diff
@@ -1,10 +1,17 @@
 {
-  "name": "sentry-cli-binary",
+  "name": "@sentry/cli",
   "version": "1.25.0",
   "description": "A command line utility to work with Sentry. https://docs.sentry.io/hosted/learn/cli/",
```

Comment:
What is the minimum supported node version? We should add an `engines` field here.